agent/balrogagent/cmd.py

```diff
@@ -43,7 +44,14 @@
                     else:
                         logging.debug("Unknown change type!")
 
-                    # If it *is* ready, enact it!
+                    # If it *is* ready, check if all the required signoffs have
+                    # been obtained
```

Comment:
This comment needs an update - "if it *is ready" doesn't make sense when that's not checked until ~8 lines later.